src/devices/bus/coco/meb_rtime.cpp

```diff
@@ -0,0 +1,173 @@
+// license:BSD-3-Clause
+// copyright-holders:tim lindner
+/***************************************************************************
+
+    meb_rtime.cpp
+
+    Real time clock for Disto mini expansion bus
+
+    Includes a Centronics parallel port interface
+
+***************************************************************************/
+
+#include "emu.h"
+#include "meb_rtime.h"
+#include "machine/msm6242.h"
+#include "bus/centronics/ctronics.h"
+
+#include "meb_intrf.h"
+
+#define RTC_TAG "rtc"
+#define CENTRONICS_TAG "centronics"
+#define LATCH_TAG "latch"
```

Comment:
Using macros for this stuff when you don’t need to use the literal tag in multiple places just makes the code harder to read.

Reply:
Fixed.